main/auto-updater.js

```diff
@@ -29,3 +31,22 @@ function init () {
     log('App update downloaded: ', releaseName, updateURL)
   })
 }
+
+function checkForUpdates () {
+  // Electron's built-in auto updater only supports Mac and Windows, for now
+  if (process.platform !== 'linux') {
+    return autoUpdater.checkForUpdates()
+  }
+
+  // If we're on Linux, we have to do it ourselves
+  get.concat(config.AUTO_UPDATE_URL, function (err, res, data) {
+    if (err) return log('Error checking for app update: ' + err.message)
+    if (![200, 204].includes(res.statusCode)) return log('Error checking for app update, got HTTP ' + res.statusCode)
+    if (res.statusCode !== 200) return
+
+    var obj = JSON.parse(data)
+    // TODO: version should be included in the response object, we shouldn'v have to parse obj.name
```

Comment:
I just updated the endpoint for you: https://webtorrent.io/app/update?version=0.0.0

Can you update the PR?